src/utilities/renderWithTooltip.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Icon, Tooltip } from 'antd';
+
+export function renderWithTooltip (label: string | null, tooltip: string) {
```

Comment:
Could easily be a `<ToolTip />` component that takes a prop to show tooltip or not.